linux_os/guide/system/accounts/accounts-restrictions/password_storage/no_empty_passwords_etc_shadow/rule.yml

```diff
@@ -13,6 +13,10 @@ description: |-
     <pre>$ sudo passwd [username]</pre>
     Lock an account:
     <pre>$ sudo passwd -l [username]</pre>
+    Note that this rule is not applicable for systems running within a
+    container. Having user with empty password within a container is not
+    considered a risk, because it should not be possible to directly login into
+    a container anyway.
```

Comment:
maybe a warning is better in this context. Something like: https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/software/sudo/sudoers_no_root_target/rule.yml#L42-L43

Reply:
Okay, I will change it also in `no_empty_passwords` rule